include/torch-mlir/Dialect/Torch/IR/TorchOps.td

```diff
@@ -1165,4 +1165,41 @@ def Torch_ShapeCalculateYieldShapesOp : Torch_Op<"shape.calculate.yield.shapes",
   let hasVerifier = 1;
 }
 
+def Torch_StoreOp : Torch_Op<"store", []> {
+  let summary = "store operation";
+  let description = [{
+    This op represents a prim::Store node in the Python object graph.
+  }];
+  let arguments = (ins StrAttr:$var, AnyTorchType:$value);
+  let assemblyFormat = [{ $var `,` $value attr-dict `:` qualified(type($value)) }];
+}
+
+def Torch_LoadOp : Torch_Op<"load", []> {
```

Comment:
please follow the naming convention -- in this case, it would be prim::Load is `torch.prim.load`

Reply:
Thanks!  Fixed by renaming it to `torch.prim.Load` (so that we can reuse some of the existing cases) and moved the ops to be grouped together with other interpreter builtin ops.